build/Makefile

```diff
@@ -354,7 +357,7 @@ uninstall: $(ensure-build-image)
 
 # Build a static binary for the gameserver controller
 build-controller-binary: $(ensure-build-image)
-	$(GO_BUILD_LINUX_AMD64) \
+	$(GO_BUILD_LINUX_$(ARCH)) \
```

Comment:
Same comment as above - would like to see a WITH_ARM64 variable that when enabled, also builds the arm64 binary - rather than a single run build with a single arch.